InterviewFlashCards/QueryManager.swift

```diff
@@ -1,12 +1,11 @@
-
 import Foundation
 import Firebase
 import FirebaseDatabase
 
-public typealias JSON = [[String: Any]]
-public protocol Networking {
+public typealias JSON = [String: Any]
```

Comment:
whatever code is highlighted in red is the old code, right? 

like you added `+public typealias JSON = [String: Any]` and that's why it's in green and has the +? so didn't you change `JSON` from an array off dictionaries to just be a dictionary?